scripts/install-ksops-archive.sh

```diff
@@ -4,7 +4,7 @@ set -e
 # Require $XDG_CONFIG_HOME to be set
 if [[ -z "$XDG_CONFIG_HOME" ]]; then
   echo "You must define XDG_CONFIG_HOME to use a kustomize plugin"
-  echo "Add 'export XDG_CONFIG_HOME=\$HOME/.config' to your .bashrc or .zshrc"
+  echo "Add 'export XDG_CONFIG_HOME=$HOME/.config' to your .bashrc or .zshrc"
```

Comment:
Hi @Leen15 👋  thanks for making a PR! 

The `\` in the message is intentional. On MacOS/Linux, without the `\` the command will evaluate the `$HOME` environment variable `/my/home/.config/`, but it's best practice to include the `$HOME` variable rather than the value of it when adding to your `rc` file. 

What operating system are you using? Did you run into this issue when copy-pasting from this script directly rather than the output of the script in your shell?